doc/build-openbsd.md

```diff
@@ -1,105 +1,57 @@
 OpenBSD build guide
 ======================
-(updated for OpenBSD 6.0)
+(updated for OpenBSD 6.9)
 
-This guide describes how to build gridcoinresearchd and command-line utilities on OpenBSD.
-
-As OpenBSD is most common as a server OS, we will not bother with the GUI.
+This guide describes how to build gridcoinresearchd, command-line utilities, and GUI on OpenBSD.
 
 Preparation
 -------------
 
 Run the following as root to install the base dependencies for building:
 
 ```bash
-pkg_add gmake libtool libevent
-pkg_add autoconf # (select highest version, e.g. 2.69)
-pkg_add automake # (select highest version, e.g. 1.15)
-pkg_add python # (select highest version, e.g. 3.6)
+pkg_add git gmake libtool libevent boost libzip
+pkg_add qt5 libqrencode # (optional for enabling the GUI)
+pkg_add autoconf # (select highest version, e.g. 2.71)
+pkg_add automake # (select highest version, e.g. 1.16)
+pkg_add python # (select highest version, e.g. 3.9)
```

Comment:
should be 3.10 now

Reply:
Let me spin up an OpenBSD 7.0 vm to see if it still works.

Reply:
Yeah no, it's terribly out of date upstream.

Is it worth putting the effort in fixing it?

Reply:
Anyway I just checked and OpenBSD 7.0 does not package python 3.10.